src/types/Venue.ts

```diff
@@ -35,6 +36,7 @@ export interface Venue {
   parentId?: string;
   template: VenueTemplate;
   name: string;
+  access?: VenueAccessMode;
```

Comment:
Inferring from some of the usages of this field, and how it uses `.include`, it seems as though a lot of the code might expect this to be an array as it used to be? Not sure. But worth checking/aligning one way or the other.

Reply:
The rest of the code is changed now and considers this field a string.